components/tag/src/tag-list/tag-list.ts

```diff
@@ -166,13 +170,42 @@ export class DtTagList implements AfterContentInit, OnDestroy {
     this._setWrapperBoundingProperties(true);
   }
 
+  /** @internal Evaluates the width of the more button */
+  _handleMoreButtonProperties(_hiddenTagCount: number): void {
+    if (_hiddenTagCount) {
+      // Since the font-variant is set to tabular-nums we can use constants for the width
+      _hiddenTagCount < 10
+        ? this._setMoreBtnBoundingProperties(58)
+        : _hiddenTagCount < 100
+        ? this._setMoreBtnBoundingProperties(65.5)
+        : _hiddenTagCount < 1000
+        ? this._setMoreBtnBoundingProperties(73)
+        : _hiddenTagCount < 10000
+        ? this._setMoreBtnBoundingProperties(80.5)
+        : this._setMoreBtnBoundingProperties(88);
```

Comment:
Can we try to get rid of these magic numbers or describe why this is needed? Furthermore, .5 pixel values can't really be rendered so we should not set them. 

I've also tested the preview and I encountered some weird cases:
The first 2 were encountered after changing the width of the demo.
![image](https://user-images.githubusercontent.com/2905693/73814629-eac2ba80-47e3-11ea-8e4f-a1dfcad93790.png)
![image](https://user-images.githubusercontent.com/2905693/73814670-0af27980-47e4-11ea-98ca-2e8902dd66b6.png)
The next one is on initial load for me on Chrome 79
![image](https://user-images.githubusercontent.com/2905693/73814728-32e1dd00-47e4-11ea-8f40-b0aa9f622e0d.png)